axlearn/experiments/text/gpt/fuji.py

```diff
@@ -140,6 +140,29 @@ def get_trainer_kwargs(model_size: str, *, vocab_size: int, version: Version) ->
                 ),
             ),
         )
+    elif model_size == "simple":
```

Comment:
Thanks! Does this need to be separate from `"test"` (which is itself intended to be the testing configuration)? 

In particular, we can configure `mesh_rules` for the accelerator that you are testing on. This way, it'll run on both CPU and the target testing hardware.

The only other differences seem to be batch sizes and eval/saving more frequently, which seem tolerable as defaults. WDYT?

Reply:
Re mesh_rules, yeah sometimes we test on v4-8, and we need something like (-1,1,4,1,1). 

However, re eval/saving/max step, we do want to have a config that terminate training early. As long as the training runs for a few thousands steps without problems, then we know the jax testing passes

Reply:
Thanks! That sounds reasonable, adding the new defaults to the "test" configuration instead for frequent saving/early termination. 
On mesh rules, I'll leave the default for it to work on CPU, but can you clarify how we can configure the rules for specific accelerators? as Maggie mentioned, we'd be testing mainly on smaller TPU shapes like v4-8.

Reply:
Thanks @markblee !
Committed the changes you suggested, it makes sense to add data=-1 to the default configuration.